### DISCUSSION TITLE
Im having a stranger error in routes and controler flux in adonis, this is a specific possible bug?

### DISCUSSION THREAD

**Discussion by sirpepsiw** [adonisjs/core#2706] [Help]
Hi everybody recently im clone a git repository in a new computer and install all dependences. The problem? im try call some routes, the server responds in status 200 but adonis not call method at controller

`Route.post('/newDepartamento', 'DepartamentsController.registrarDepartamento').middleware('auth');`

im try modify the controller waiting a error but nothing happens it's as if the controller doesn't exist. but im have the compiled proyect uploaded in heroku and this works...

In the same pc im have another 'login controller' and this is works im not have any idea im try run 'npm cache clean --force' but does not work.. before im copy the compile project and paste in the controller and solve bugs, im try putting up console.logs and crashing my code but again it's as if the controller doesn't exist... the name in the route and the controller is correctly and im try change the name of controller in the route waiting a error but noting... Any idea? this is a possible bug?

my controller method:

```
async registrarDepartamento({ request, auth, response }:HttpContextContract) {
        const validation = await schema.create({
            nombre: schema.string({}, [
                rules.required()
            ])
        });
        const messages = {
            "required": "Es necesario llenar el campo del nombre",
        };
        const validate = await request.validate({
            schema: validation,
            messages: messages
        });
        const departamento = new Departament;
        departamento.account_id = auth.user!.id;
        departamento.name = validate.nombre;
        await departamento.save();
        return response.json({
            status: 'sure',
            message: 'registrado',
            data: departamento
        });
    }
```

the link to my repository : https://github.com/gario300/ZprintBack

this same happens in `Route.post('/newMarca', 'MarcasController.registrarMarca');`

**Comment by thetutlage** [comment]
Your project dependencies are completely out of date. I recommended updating all dependencies.
